auth-web/src/views/auth/RequestNameView.vue

```diff
@@ -16,36 +16,41 @@
           <v-icon size="8" class="list-item-bullet mt-5">mdi-square</v-icon>
           <v-list-item-content>
             <v-list-item-subtitle class="list-item-text">
-              You can choose to incorporate a numbered Benefit Company
+              You can choose to incorporate a <numbered-company-tooltip />
               and start your incorporation immediately.
             </v-list-item-subtitle>
           </v-list-item-content>
         </v-list-item>
         <!-- Panel Btns -->
         <div class="request-name-info-btns mt-5">
           <v-btn large color="#003366" class="white--text"
-            href="https://www.bcregistrynames.gov.bc.ca/nro/" target="_blank" rel="noopener noreferrer">
+                 :href="nroUrl" target="_blank" rel="noopener noreferrer">
             Request a Name
           </v-btn>
           <p class="mt-5">Have an existing Name Request?
-            <a href="https://www.bcregistrynames.gov.bc.ca/nro/nro.htm?_flowId=anonymous-monitor-flow&_flowExecutionKey=e1s1"
+            <a :href="`${nroUrl}nro.htm?_flowId=anonymous-monitor-flow&_flowExecutionKey=e1s1`"
```

Comment:
While this works, I suggest using a getter so it looks cleaner.

The principle I'm trying to follow is to keep "code" out of the template. This instance isn't serious enough to preempt my approval but, you know, it would be cleaner 😀

Reply:
Absolutely agree, Will update this!

Reply:
Updated!